contents/incentivization.rst

```diff
@@ -0,0 +1,152 @@
+ * accounting system
+ * file insurance
+ * litigation
+
+
+At this point we only document the accounting system. The other parts will be documented in the future, although they are specified partly already in the documentation [`Incentives system for Swarm <https://swarm-gateways.net/bzz:/swarm.eth/ethersphere/orange-papers/1/sw%5E3.pdf>`_]. Also, this documentation only is meant as an entry point for understanding and operating Swarm. For a thorough and detailed documentation the complete incentives system documentation (linked just above) should be consulted.
+
+Accounting
+==========
+At the core of Swarm's incentive system is the SWarm Accounting Protocol, or SWAP. It is a very simple protocol which relies on a tit-for-tat p2p accounting model:
+ * a node gets rewarded for serving resources
+ * a node gets charged for requesting resources
+
+Every peer maintains a local database on its own file system which persists accounting information of all peers with which it exchanged data. On every change, the in-memory accounting structures are updated and immediately persisted to disk in order to avoid synchronization issues.
+
+.. note::
+
+   SWAP excels at its simplicity. It is not the goal and would completely defeat the purpose of the system to introduce bullet-proof consistency and auditing - Swarm is not a blockchain in its own terms.`
+
+
+.. important::
+   The Swarm node operator is responsible for maintenance of the local accounting database (a LevelDB instance). Tampering with it, altering its contents or removing it altogether may result in the node becoming inoperable.
+
+If a node requests data from a peer, when the peer delivers the data, the peer credits an amount equivalent to the price of that message in its local database, at the index representing the node it delivers to. The node, when it receives the message (data) from the peer, debits itself in its local database the same amount at the index of the peer. 
+
+.. image:: img/swap.svg
+   :alt: Simple SWAP exchange between two nodes 
+   :width: 400
+
+In the above example, A requested data from B, which B then delivers to A (could be a chunk). Assuming a price of 200 units for the chunk, B credits itself in its local database plus 200 at the index for node A, while A, when it finally receives the data from B, debits minus 200 in its local database at the index for node B.
+
+Due to the distributed nature of Swarm nodes, entries in the local databases do not happen simultaneously, and there is no notion of transactions nor confirmations; in normal operation, both nodes account for the message with the correspondent node at the moment of the action (for the sending peer, at the moment of sending, for the receiving peer, at the moment of receiving). 
```

Comment:
> This sentence is a bit weird to me.
> 
> Maybe:
> Due to the distributed nature of Swarm, entries separate local databases are not atomic as all messages must be broadcasted over a network and processed by the local, running instances of Swarm. In normal operation, however, we assume that peers credit and debit roughly the same amounts to each others balances over time. Of course, a small variance in balance may exist, but if the nodes run the correct software and are not otherwise limited, all variance in balance is expected to be 0 in the long-run.

out of this, i would keep `in normal operation, we assume that peers credit and debit roughly the same amounts to each others balances over time. Of course, a small variance in balance may exist, but it is expected to be 0 in the long-run.`